src/horizon/server.ts

```diff
@@ -131,8 +135,10 @@ export class Server {
    *  // earlier does the trick!
    *  .build();
    * ```
-   * @argument {number} seconds Number of seconds past the current time to wait.
-   * @argument {bool} [_isRetry=false] True if this is a retry. Only set this internally!
+   * @param seconds
+   * @param {number} seconds Number of seconds past the current time to wait.
+   * @param _isRetry
+   * @param {bool} [_isRetry] True if this is a retry. Only set this internally!
```

Comment:
this is from eslint being weird, right? we should only have one `@param` per param
```suggestion
   * @param {number} seconds Number of seconds past the current time to wait.
   * @param {bool} [_isRetry=false] True if this is a retry. Only set this internally!
```